### ISSUE TITLE
npm dev:client doesn't work anymore

### ISSUE THREAD

**Issue by dedesite** [Chocobozzz/PeerTube#158]
I don't know why but `npm run dev:client` doesn't work anymore. I clean up my `node_modules` dir and rebuild everything but when I go to `localhost:3000/` I got nothing.
No errors in the console, just the regular debug print of the SQL requests. 

For now, I can work with `npm run dev:server` but it's not optimal.

**Comment by Chocobozzz** [comment]
I can't reproduce. Please tell me what you have in your browser:
 * HTML source
 * console
 * network

**Comment by dedesite** [comment]
Sorry I created this bug too quickly and didn't do a proper report.
Once I have more time, I'll do a full clean install (from a git clone) of PeerTube and fill the bug report if I still got the problem.

**Comment by bnjbvr** [comment]
Will probably be fixed by https://github.com/Chocobozzz/PeerTube/pull/177.

**Comment by bnjbvr** [comment]
@dedesite Can you try again now that #177 has been merged, please?

**Comment by dedesite** [comment]
I tried to reinstall the project from scratch but now I got some errors trying to build a native node module `rdf-canonize`. Don't have time to investigate... Maybe it's because I don't the right dependency, I'm still under Ubuntu 14.04 and that might be the problem.

If everything works for you and others, feel free to close this issue, will reopen it if I still got it with an updated system.

Thanks
